test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/apiKeys.test.js

```diff
@@ -135,101 +136,79 @@ describe('#compileApiKeys()', () => {
       ).to.equal('ApiGatewayDeploymentTest');
     });
   });
+});
+
+describe('lib/plugins/aws/package/compile/events/apiGateway/lib/apiKeys.test.js', () => {
```

Comment:
Let's move this out to a separate top-level `describe` block

Reply:
Hi @pgrzesik

I'm a bit confused, I think the `describe('lib/plugins/aws/package/compile/events/apiGateway/lib/apiKeys.test.js' ...` is already at the top-level. Am I misunderstanding what you mean?

Thanks

Reply:
Sorry - when reviewing I somehow saw it as nested in the `#compileApiKeys` `describe`, it's all good :+1: